package.json

```diff
@@ -147,6 +149,11 @@
         },
         "dir"
       ]
+    },
+    "pkg": {
+      "artifactName": "tidepool-uploader-${version}.${ext}",
+      "allowAnywhere": false,
+      "allowRootDirectory": false
```

Comment:
Yeah - I sort of understand, as you can gather from their documentation that these flags translate directly to flags in the Mac install API, but just from reading these I wouldn't know what their actual effect is.
I also agree that it seems highly unlikely that clinics would set up individual users for their computers. If that's the case, then we'll have to jump that hurdle when we get there. At least now it can successfully update.